build.gradle

```diff
@@ -468,6 +468,79 @@ project(':iceberg-mr') {
   }
 }
 
+if (jdkVersion == '8') {
+  project(':iceberg-mr-hive3') {
```

Comment:
Can we rename this to `:iceberg-hive3`? I don't think that we need the `mr` part because this doesn't really contain MR classes.

Reply:
sure, renamed it